tests/hpc/slurm_master.pm

```diff
@@ -189,8 +200,9 @@ sub t08_basic() {
     my $name = 'pdsh-slurm';
     my $description = 'Basic check of pdsh-slurm over ssh';
     my $result = 0;
-
-    zypper_call('in pdsh pdsh-slurm');
+    # $slurm_pkg-munge is installed explicitly since slurm_23_02
```

Comment:
I think this comment is not need. It's a feature not bug. If you mention bug reference, one would think thare is something pending, but there is nothing pending in this case.

Reply:
I thought to add it, to give awareness of the change. This change the behavior of the test if neglected, and to avoid reviewers checking again what is the difference. Another option is to add the line on the description. wdyt?

Reply:
personally I would opt for having this comment; maybe amending line 204 to: "see this bug for more technical details: bsc1214094" could be a solution to @czerw right reservations  (If you mention bug reference, one would think thare is something pending)?

Reply:
yes, lets keep first comment, just remove bug reference

Reply:
Done and ready for review